WordPress/Sniffs/VIP/SlowDBQuerySniff.php

```diff
@@ -50,6 +51,10 @@ public function getGroups() {
 	 */
 	public function process_token( $stackPtr ) {
 
+		if ( $this->has_whitelist_comment( 'slow query', $stackPtr ) ) {
+			return;
+		}
+
 		if ( $this->has_whitelist_comment( 'tax_query', $stackPtr ) ) {
 			return;
```

Comment:
@barryceelen Hang on... just thinking.. the sniff probably gets triggered twice on that same snippet of code, causing the double warning.
To get round that, we'd need to return the token to the whitelist comment to skip past the other tokens on that line.
As it's within an array assignment, you can't use the `$phpcs->findEndOfStatement()` method, so we may need to look into changing the whitelist check method to return `false` when no whitelist comment is found (like it already does) and the `int` token pointer to the whitelist comment in case a whitelist comment is found which could then be passed via return.

Mind: this is just some random free thinking about it without looking at the code, so there may well be a simpler way to do this.